docs/guides/transaction_request_flow.md

```diff
@@ -0,0 +1,110 @@
+# Transaction Request Flow
+
+## Introduction
+
+Transaction requests are a way to create pre-transactions with only one side (either sender or receiver). That pre-transaction will become a full transaction once the other side has consumed it.
+
+
+Here is a quick example to give you a better idea of how it works:
+
+1. Alice generates a transaction request to receive 10 OMG. The transaction ID can be embedded in a QR code to provide a better user experience.
+2. Alice shows her QR code to Bob.
+3. Bob scans it, see the transaction request, updates some parts of it if needed, finalizes it and sends 10 OMG to Alice.
+
+That's the simple version. Transaction requests come with a bunch of options to make them as flexible as possible.
+
+## A look at transaction requests
+
+When creating transaction requests, a certain number of fields are optional but allow the configuration of the request. Walking through those fields is a good way to get a better understanding.
+
+See [the Swagger doc](https://ewallet.demo.omisego.io/api/docs.ui#/TransactionRequest/transaction_request_create) for more details.
+
+Here is the model attributes when creating transaction requests:
+
+```
+type:
+  type: string
+  enum:
+    - send
+    - receive
+token_id:
+  type: string
+  description: "The minted token ID to use for the transaction."
+amount:
+  type: integer
+  default: null
+  description:
+    "The amount to transfer. If not specified, the consumption will need to set
+     the amount."
+correlation_id:
+  type: string
+  default: null
+  description: "Optional unique identifier to tie a request with your system."
+account_id:
+  type: string
+  default: The owner of the given address.
+provider_user_id:
+  type: string
+  default: The owner of the given address.
+address:
+  type: string
+  description:
+    "If not specified, the current user's primary balance will be used.
+     If specified, it needs to be one of the account's or user's addresses."
+require_confirmation:
+  type: boolean
+  default: false
+  description:
+    "Indicates if a consumption of the created request needs to be approved before
+    being processed."
+max_consumptions:
+  type: integer
+  default: null
+  description:
+    "The number of times this created request can be consumed."
+consumption_lifetime:
+  type: integer
+  default: null
+  description:
+    "The lifetime in milliseconds of how long a consumption can stay
+     'pending' before being expired. If a consumption is not approved before its
+     expiration date, it will become invalid and be cancelled. This property
+     can be used to avoid stale consumptions blocking the 'max_consumptions' count."
+expiration_date:
+  type: string
+  default: null
+  description:
+    "The datetime at which the created request should expire (no one will be
+     able to consume it anymore). The format is yyyy-MM-dd'T'HH:mm:ssZZZZZ."
```

Comment:
Nice. I like that the format is written, not the ISO code so users don't need to lookup.